source/Calamari.Common/Variables/SpecialVariables.cs

```diff
@@ -0,0 +1,26 @@
+namespace Calamari.Common.Variables
+{
+    public static class SpecialVariables
+    {
+        
+        public static readonly string CalamariWorkingDirectory = "OctopusCalamariWorkingDirectory";
+        public static readonly string RetentionPolicySet = "OctopusRetentionPolicySet";
+        public static readonly string PrintVariables = "OctopusPrintVariables";
+        public static readonly string PrintEvaluatedVariables = "OctopusPrintEvaluatedVariables";
+        
+        // Everything below here needs a review
+        public static readonly string CopyWorkingDirectoryIncludingKeyTo = "Octopus.Calamari.CopyWorkingDirectoryIncludingKeyTo";
+        public static readonly string SkipFreeDiskSpaceCheck = "OctopusSkipFreeDiskSpaceCheck";
+        public static readonly string FreeDiskSpaceOverrideInMegaBytes = "OctopusFreeDiskSpaceOverrideInMegaBytes";
```

Comment:
@slewis74 What do you think about just inlining variables that are only used in one place?

Reply:
I'd be ok with moving them to where they are used, i.e. they just live in the command's implementation or something like that.

Reply:
I think the habit is to push everything up here because of our approval tests that make sure SpecialVariables doesn't change without consideration. The tradeoff is that when everything is special, nothing is special.

Reply:
Do we have those approval tests in Calamari? I didn't see an approval file change in this PR